scripts/gdocs_reader.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Imports to access gdocs
+import pickle
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+# Imports for parsing yaml
+import os
+import typing as t
+from robust import yaml
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+
+# The ID and range of robust spreadsheet.
+SPREADSHEET_ID = '1WMnp8a2I79aVBS4x_vjAkxFzzELsBpmjYgMEuytZOhg'
+RANGE_NAME = 'Coding!A2:F'
+
+dir_here = os.path.dirname(__file__)
+token_path = os.path.join(dir_here, 'token.pickle')
+credentials_path = os.path.join(dir_here, 'credentials.json')
+
+def get_credentials():
+    creds = None
+    # If there are no (valid) credentials available, let the user log in.
+    if os.path.exists(token_path):
+        with open(token_path, 'rb') as token:
+            creds = pickle.load(token)
+
+    if not creds or not creds.valid:
+        if creds and creds.expired and creds.refresh_token:
+            creds.refresh(Request())
+        else:
+            flow = InstalledAppFlow.from_client_secrets_file(
+                credentials_path, SCOPES)
+            creds = flow.run_local_server(port=0)
+        # Save the credentials for the next run
+        with open(token_path, 'wb') as token:
+            pickle.dump(creds, token)
+    return creds
+
+def get_gsheets_data() -> t.Iterator[t.List[str]]:
+    creds = get_credentials()
+
+    service = build('sheets', 'v4', credentials=creds)
+
+    # Call the Sheets API
+    sheet = service.spreadsheets()
+    result = sheet.values().get(spreadsheetId=SPREADSHEET_ID,
+                                range=RANGE_NAME).execute()
+    values = result.get('values', [])
+
+    if not values:
+        print('Error: No data found.')
+    else:
+        for row in values:
```

Comment:
How does this deal with the header row?

Reply:
The range starts from `A2` so it is ignored.